Techome/Techome/Views/AddCaffeineDetail/AddCaffeineDetailView.swift

```diff
@@ -0,0 +1,390 @@
+                    Text("5")
+                        .font(.title2)
+                        .fontWeight(.bold)
+                        .foregroundColor(.primaryBrown)
+                    Text("시")
+                        .font(.footnote)
+                        .foregroundColor(.secondaryTextGray)
+                        .padding(.trailing, AddCaffeineDetailViewLayoutValue.Paddings.AddCaffeineAmountToUnitPadding)
+                    Text("33")
+                        .font(.title2)
+                        .fontWeight(.bold)
+                        .foregroundColor(.primaryBrown)
+                    Text("분")
+                        .font(.footnote)
+                        .foregroundColor(.secondaryTextGray)
+                }
+            }
+        }
+        .padding(.vertical, AddCaffeineDetailViewLayoutValue.Paddings.effectCardInsideVerticalPadding)
+        .padding(.horizontal, AddCaffeineDetailViewLayoutValue.Paddings.effectCardInsideHorizontalPadding)
+    }
+}
+
+struct EffectSectionBackground: View {
+    var body: some View {
+        RoundedRectangle(cornerRadius: AddCaffeineDetailViewLayoutValue.CornerRadius.cardRadius)
+            .foregroundColor(.white)
+            .shadow(color: .primaryShadowGray, radius: AddCaffeineDetailViewLayoutValue.CornerRadius.shadowRadius, x: .zero, y: .zero)
+    }
+}
+
+struct AddCaffeineButton: View {
+    var body: some View {
+        Button {
+            
+        } label: {
+            Text("추가하기")
+                .font(.title2)
+                .fontWeight(.bold)
+                .foregroundColor(.backgroundCream)
+                .background(AddCaffeineButtonBackground())
+        }
+        
+        
+    }
+}
+
+struct AddCaffeineButtonBackground: View {
+    var body: some View {
+        RoundedRectangle(cornerRadius: AddCaffeineDetailViewLayoutValue.CornerRadius.cardRadius)
+            .foregroundColor(.primaryBrown)
+            .padding(.horizontal, AddCaffeineDetailViewLayoutValue.Paddings.cardVerticalPadding)
+            .frame(width: UIScreen.main.bounds.width, height: AddCaffeineDetailViewLayoutValue.Sizes.addCaffeineButtonHeight, alignment: .center)
+    }
+}
+
+//https://serialcoder.dev/text-tutorials/swiftui/rounding-specific-corners-in-swiftui-views/
+struct RoundedCornersShape: Shape {
+    let corners: UIRectCorner
+    let radius: CGFloat
+    
+    func path(in rect: CGRect) -> Path {
+        let path = UIBezierPath(roundedRect: rect,
+                                byRoundingCorners: corners,
+                                cornerRadii: CGSize(width: radius, height: radius))
+        return Path(path.cgPath)
+    }
+}
+
+struct EspressoShotCountStepper: View {
+    //TODO: 임시 더미 데이터
+    @State var shotCount: Int = 2
+    
+    var body: some View {
+        HStack(alignment: .center, spacing: .zero) {
+            Button {
+                if shotCount > 0 {
+                    shotCount -= 1
+                }
```

Comment:
```suggestion
                shotCount -= shoutCount > 0 ? 1 : 0
```

컴파일이 될,,,지는 모르겠지만 된다면 한 줄로 줄일 수 있을 것 같아요 .. !